Pipfile

```diff
@@ -22,7 +22,7 @@ maya = "*"
 flask = "*"
 flask_sqlalchemy = "*"
 # Third-Party Ethereum
-py-evm = "==0.2.0a39"
+py-evm = ">=0.2.0a39"
```

Comment:
Does it really work with this newer version? I think, there were issues with `eth-tester`?

Reply:
I think it remained locked at `0.2.0a39`

Reply:
Sure... But here in Pipfile it's updated, and then `--skip-lock` will install 0.2.0a42 or whatever, and eth-tester will fail?
Unless they shipped a newer version

Reply:
I think with the presence of a `Pipfile.lock` - there is no need to use `--skip-lock` and the best way to install is `pipenv sync`, although I think `pipenv install` will have a similar effect

Reply:
Well, at my machine, it tries to relock for some reason, and that happens forever.
Why not keep it ==0.2.0a39 as long as it is not known to work at higher versions?

Reply:
(until eth-tester gets updated)

Reply:
upd: Pipfile.lock works indeed. But still, needs to be workable even with --skip-lock

Reply:
Reverted - :heavy_check_mark: